### PR TITLE
fix: Correct typo, improve createRadio() method description, and simplify input tag

### PR DESCRIPTION
**Pull request by Dhanush111**
Related Issue
Originally reported in p5.js-website repository: [#595 - Misspelling in Docs: Reference > createRadio()](https://github.com/processing/p5.js-website/issues/595)

Summary

- Fixed a typo in the documentation: changed "paremeter" to "parameter" in the description of the `myRadio.option()` method.
- Updated the description of the `myRadio.disable()` method for better clarity:
  - Changed "enables the entire radio button if true is passed and disables it if false is passed."
    to "Disables the radio button if true is passed, and enables it if false is passed."
- Simplified the input tag in the documentation:
  - Changed `<input name="food"></input>` to `<input name="food">` since `<input>` is self-closing.

Motivation
These updates improve the clarity, accuracy, and HTML standards compliance of the createRadio documentation, making it easier for developers to understand and implement this method.

Additional Notes
Please review the changes to ensure they align with the documentation style and standards of the project.

### CODE REVIEW

**Comment by Dhanush111** [comment]
@osteele @andrewn @zeke @bulkan Could you please review this PR? All required changes have been made, and I’m awaiting your feedback for the merge.

**Comment by bulkan** [comment]
@Dhanush111 not sure why you tagged me in this issue as I'm not part of the collaborators here?

**Review by perminder-17 — COMMENTED** [review]
Since you're working on fixing the createRadio() method, could you also adjust the examples? Currently, the options are not displaying correctly. 

image

image


I believe this can be resolved by applying some styling to myRadio, perhaps by setting `display: flex` and `flex-direction: column`.

File: src/dom/dom.js [hunk 1]
```diff
@@ -1442,18 +1442,18 @@ p5.prototype.createSelect = function(...args) {
...
+ * - `myRadio.option(value, [label])` adds an option to the menu. The first parameter, `value`, is a string that sets the option's value and label. The second parameter, `label`, is optional. If provided, it sets the label displayed for the `value`. If an option with `value` already exists, its label is changed and its value is returned.
  * - `myRadio.value()` returns the currently-selected option's value.
  * - `myRadio.selected()` returns the currently-selected option.
  * - `myRadio.selected(value)` selects the given option and returns it as an <a href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement" target="_blank">`HTMLInputElement`</a>.
- * - `myRadio.disable(shouldDisable)` enables the entire radio button if `true` is passed and disables it if `false` is passed.
+ * - `myRadio.disable(shouldDisable)` Disables the radio button if `true` is passed, and enables it if `false` is passed.
```
[the remark this answers is not in the document]
Comment (Dhanush111) [reply src/dom/dom.js:1456]: @perminder-17 Yes, i too saw those things were not right, the thing is, I’m having trouble running it locally or on GitHub Pages right now, so I can’t make those changes myself at the moment.

**Comment by Dhanush111** [comment]
@perminder-17  Yes, i too saw those things were not right, the thing is, I’m having trouble running it locally or on GitHub Pages right now, so I can’t make those changes myself at the moment.

File: .github/workflows/jekyll-gh-pages.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
```
Comment (perminder-17) [inline .github/workflows/jekyll-gh-pages.yml:2]: Sorry if I am unable to understand, but can you help me knowing what's the use of adding this file?

**Review by perminder-17 — COMMENTED** [review]
No problem, I will open up a PR on fixing this very soon.

File: .github/workflows/jekyll-gh-pages.yml (hunk 2 above)
Comment (Dhanush111) [reply .github/workflows/jekyll-gh-pages.yml]: @perminder-17 sorry, it was accidental(tried to build it on github using that file)

**Review by perminder-17 — APPROVED** [review]
Sorry for the delay in review, looks good to me.

**Review by davepagurek — APPROVED** [review]

**Pull request merged**